app/views/unattended/provisioning_templates/snippet/kickstart_rhsm.erb

```diff
@@ -0,0 +1,30 @@
+<%#
+kind: snippet
+name: kickstart_rhsm
+model: ProvisioningTemplate
+snippet: true
+description: |
+  Generate Anaconda RHSM configuration for Red Hat OS registration
+test_on:
+- rhel9_dhcp
+-%>
+<%
+  subman_keys = host_param('kt_activation_keys') || host_param('activation_key')
+  subman_org = host_param('subscription_manager_org')
+  subman_registration = host_param_true?('subscription_manager') || subman_keys
```

Comment:
if I'm trying to override global param "rhel9" value with host param being an empty string, this would still resolve to true right? should we use subman_keys.presence?

Reply:
the similar logic should probably apply also below, perhaps it's an edge case

Reply:
`global param "rhel9" ` you mean `kt_activation_keys` or `activation_key`?

Reply:
Imagine subscription_manager is empty/false then it falls back to subman_keys, which can be either of the two you mentioned, but I think their value can be an empty string (not tested) which is considered truthy in Ruby. I think this can be tested and potentially addressed separately from the organization selection issue.

Reply:
Let's move the discussion to https://github.com/theforeman/foreman/pull/9425.